docs/guides/platforms/gke/index.md

```diff
@@ -607,14 +610,38 @@ stash-backup-sample-mariadb-backup   */5 * * * *   False     0        2m59s
 
 Here, `False` in the `SUSPEND` column means the CronJob is no longer suspended and will trigger in the next schedule.
 
+## Allow Operator to List Snapshots
+
+Stash operator uses its own `ServiceAccount` to list the snapshots from the backend. Therefore, this `ServiceAccount` should be binded with the IAM service account as well.  Run the following command to get the service account used by the Stash operator,
```

Comment:
When you list Snapshots using `kubectl get snapshot` command, Stash operator itself read the Snapshots directly from the backend. So, the operator needs permission to access the bucket.

Stash operator has it own's `ServiceAccount`. Therefore, this operator `ServiceAccount` should be ...